lib/protected_attributes/version.rb

```diff
@@ -1,3 +1,3 @@
 module ProtectedAttributes
-  VERSION = "1.7.0".freeze
+  VERSION = "1.8.0".freeze
```

Comment:
If you could revert this line that would be great. In the future Id recommend you do not change the version numbers in a PR.

Reply:
Thanks! I'll keep that in mind.